src/Symfony/Component/PropertyAccess/Annotation/Adder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Property accessor adder configuration annotation.
+ *
+ * @Annotation
+ * @Target({"METHOD"})
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+class Adder
```

Comment:
Btw you should make it possible to omit `property` when using the annotation:

~~~php
/**
 * @PropertyAdder("fields")
 */
public function addFieldToList($field)
{
    // ...
}
~~~